### DISCUSSION TITLE
Is there a reason that otelzap doesn't use the passed message as the event name?

### DISCUSSION THREAD

**Discussion by kklipsch** [uptrace/opentelemetry-go-extra#77] [Q&A] [answered]
https://github.com/uptrace/opentelemetry-go-extra/blob/main/otelzap/otelzap.go#L201

Uses a generic 'log' message.  But the passed in user msg is available.  Is this intentional? And if so why?

**Comment by vmihailenco** [comment, marked as answer]
Because event names must have low cardinality which can't be guaranteed if we use error messages - see https://uptrace.dev/opentelemetry/span-naming.html#span-names
